docs/usage.md

```diff
@@ -184,7 +182,7 @@ PSM fdr threshold to align peptide ids with reference run (default = 0.01)
 
 ### `--unimod`
 
-Path to unimod.xml file describing modifications (https://github.com/nf-core/test-datasets/tree/diaproteomics/unimod.xml)
+Path to unimod.xml file describing modifications ("https://github.com/nf-core/test-datasets/tree/diaproteomics/unimod.xml")
```

Comment:
does this need to link to the raw data? this might link to an HTML representation of the file. did not check though